.travis.yml

```diff
@@ -1,31 +1,33 @@
 language: ruby
+sudo: false
 
 rvm:
-  - 2.0.0
-  - 1.9.3
+  - 2.3.1
```

Comment:
I think `2.4.0-preview1` needs to be here? There were no 2.4 tests run: https://travis-ci.org/Shopify/shopify_api/builds/142531600